### DISCUSSION TITLE
Seams product attribute search incomplete

### DISCUSSION THREAD

**Discussion by zssvaidar** [atrocore/atrocore#687] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/30953478/231346325-a9332048-1509-4c69-8580-720964a47792.png)

When using pim 1.6.*, 1.7.*  or above the search of product by attribute in products page and export screen doesm't work. It is the case for demo.atropim.com website also

**Comment by rratsun** [comment]
ok. We have created an internal issue and will solve it soon.

**Reply by rratsun** [reply, marked as answer]
Hello,

The problem solved at PIM 1.8.3
